src/main/java/com/sngular/kloadgen/common/tools/MapperUtil.java

```diff
@@ -0,0 +1,135 @@
+/*
+ *  This Source Code Form is subject to the terms of the Mozilla Public
+ *  * License, v. 2.0. If a copy of the MPL was not distributed with this
+ *  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.sngular.kloadgen.common.tools;
+
+import java.util.Objects;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.lang3.StringUtils;
+
+public class MapperUtil {
+
+  public static final String INTEGER = "integer";
+
+  public static final String DOUBLE = "double";
+
+  public static final String FLOAT = "float";
+
+  public static final String NUMBER = "number";
+
+  public static final String INT_64 = "int64";
+
+  public static final String LONG = "long";
```

Comment:
Maybe include all the properties like this into the `TypeConstant` class could be better